pandas/tests/indexes/test_setops.py

```diff
@@ -0,0 +1,79 @@
+'''
+The tests in this package are to ensure the proper resultant dtypes of
+set operations.
+'''
+import itertools as it
+import numpy as np
+import pytest
+
+import pandas as pd
+import pandas.util.testing as tm
+from pandas.core.dtypes.dtypes import (
+    PeriodDtype, CategoricalDtype, IntervalDtype)
+
+
+def makeEmptyIndex(_=None):
+    return pd.Index([])
+
+
+INDEXES = dict(
```

Comment:
we already have fixtures in pandas/tests/indexes/conftest.py pls use them instead of creating new ones like this. you *may* need to create derived fixtures which is ok.

Reply:
The existing fixture used pre-instantiated indexes. Added a fixture for the index factories.